src/Membership/AdherentManager.php

```diff
@@ -48,4 +48,11 @@ public function countMembersByMonthManagedBy(Adherent $referent, int $months = 6
 
         return $countByMonth;
     }
+
+    public function isAdherent(string $email): bool
```

Comment:
unused?

Reply:
@ottaviano since you changed the way we get the isAdherent column for the list, yes.